package.json

```diff
@@ -103,6 +104,7 @@
     ],
     "license": "Apache License 2.0",
     "scripts": {
+        "analyze-bundle": "GENERATE_JSON_STATS=true gatsby build && npx webpack-bundle-analyzer ./public/stats.json",
```

Comment:
where does this save generated reports?
does it generate HTML file like before?

Reply:
it does not generate the file at the moment, it just opens a server with the actual report. will change it to generate the file as well.